buildSrc/src/software/aws/toolkits/gradle/resources/ValidateMessages.kt

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.gradle.resources
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.GradleException
+import org.gradle.api.logging.Logger
+import org.gradle.api.logging.Logging
+import org.gradle.api.tasks.InputFiles
+import org.gradle.api.tasks.TaskAction
+import org.gradle.language.base.plugins.LifecycleBasePlugin.VERIFICATION_GROUP
+import java.io.File
+
+open class ValidateMessages : DefaultTask() {
+    @InputFiles
+    lateinit var paths: List<File>
```

Comment:
You can avoid lateineit using `project.objects.directoryProperty()`

Reply:
Changed (to listProperty)